apiml-sample-extension/README.md

```diff
@@ -0,0 +1,10 @@
+# API ML sample extension
+
+This is an API ML sample extension. It only contains a simple controller for testing.
+The extension is added to the API Gateway class path.
+
+## Usage
+
+If the extension is correctly added to the API Gateway classpath, it will be possible to 
+call the `https://<hostname>:<gatewayPort>/api/v1/greeting` endpoint.
```

Comment:
I would add an example of the correct call. How do you verify and what should you expect.